lib_src/lib_build_info.cmake

```diff
@@ -0,0 +1,22 @@
+set(LIB_NAME lib_src)
+set(LIB_VERSION 2.4.0)
+
+set(LIB_DEPENDENT_MODULES "lib_logging(3.1.1)"
```

Comment:
These dependencies should be specified without version numbers so that the head of develop is used because these tagged versions are before XCommon CMake was added. Once lib_logging and lib_xassert are released with XCommon CMake support, the versions can be pinned here.

Reply:
Excellent point!